wiki/Tournaments/RDOC/RDOC_2018/en.md

```diff
@@ -0,0 +1,455 @@
+- DoubleTime
+  - [FELT - Little Nova \[Opening\]](https://osu.ppy.sh/beatmapsets/344776#osu/761541)
+  - [HOUJIROU - Maihime ~buki~ \[Hyper\]](https://osu.ppy.sh/beatmapsets/28765#osu/98943)
+  - [ChouCho - My dear friend \[Insane\]](https://osu.ppy.sh/beatmapsets/206079#osu/486131)
+- FreeMod
+  - [yuikonnu - Junjou Skirt \[Insane\]](https://osu.ppy.sh/beatmapsets/200230#osu/474657)
+  - [Lon - MATRYOSHKA \[Extra\]](https://osu.ppy.sh/beatmapsets/109185#osu/285086)
+  - [IOSYS - Captain Murasa's Ass Anchor \[Ketsu Ankaa\]](https://osu.ppy.sh/beatmapsets/13422#osu/49866)
+- Tiebreaker
+  - **[sasakure.UK - ChRoNiClESeVeN feat. Annabel \[Flora\]](https://osu.ppy.sh/beatmapsets/775590#osu/1629799)**
+
+### Group Stage
+
+- NoMod
+  - [Yonezawa Madoka - Responsibility Response \[Pathetique\]](https://osu.ppy.sh/beatmapsets/276997#osu/706817)
+  - [Lime - Renai Syndrome \[Cherry Blossom's Extra\]](https://osu.ppy.sh/beatmapsets/166062#osu/905684)
+  - [Lite Show Magic (t+pazolite vs C-Show) - Crack Traxxxx \[Pantsu's Another\]](https://osu.ppy.sh/beatmapsets/139525#osu/404440)
+  - [Sota Fujimori - DANCE ALL NIGHT \[-kevincela-'s Extra\]](https://osu.ppy.sh/beatmapsets/600303#osu/1594975)
+  - [Nanahoshi Kangengakudan - anoko dokonoko \[Rizen's Extra\]](https://osu.ppy.sh/beatmapsets/654452#osu/1437419)
+- Hidden
+  - [Himeringo - Yotsuya-san ni Yoroshiku \[Skystar's Extra\]](https://osu.ppy.sh/beatmapsets/100049#osu/351996)
+  - [Noboru feat.Mitani Nana - Shiroi Yuki no Princess wa \[Extra\]](https://osu.ppy.sh/beatmapsets/439609#osu/949736)
+  - [Zektbach - Turii \~Panta rhei\~ \[Insane\]](https://osu.ppy.sh/beatmapsets/800638#osu/1680864)
+- HardRock
+  - [LeaF - Wizdomiot \[Another\]](https://osu.ppy.sh/beatmapsets/336099#osu/754799)
+  - [Nanahoshi Kangengakudan - IMAGINARY LIKE THE JUSTICE \[Frostings' Insane\]](https://osu.ppy.sh/beatmapsets/210346#osu/508558)
+  - [Xi - Majotachi no Butoukai ~ Magus \[Extra Stage\]](https://osu.ppy.sh/beatmapsets/43862#osu/158309)
+- DoubleTime
+  - [3L - Macrophylla Parasol \[Lunatic\]](https://osu.ppy.sh/beatmapsets/483344#osu/1489084)
+  - [Giga-P - Electric Angel \[Insane\]](https://osu.ppy.sh/beatmapsets/146794#osu/363639)
+  - [Zektbach - ZETA \~Sosuu no Sekai to Chouetsu sha\~ \[Hard\]](https://osu.ppy.sh/beatmapsets/34349#osu/111685)
+- FreeMod
+  - [Hatsuki Yura - Shoujo to Ougonryuu no Monogatari \[Collab Insane\]](https://osu.ppy.sh/beatmapsets/719158#osu/1542952)
+  - [Rise Against - The Eco-Terrorist In Me \[Expert\]](https://osu.ppy.sh/beatmapsets/358609#osu/819082)
+  - [kors k feat. Rie - Rising in the Sun(original mix) \[Another\]](https://osu.ppy.sh/beatmapsets/26742#osu/90024)
+- Tiebreaker
+  - **[An - Dive Archive \[Inundation\]](https://osu.ppy.sh/beatmapsets/761883#osu/1601943)**
+
+  ------------------------------------------------------------------------
+
+  ## Match Results
+
+  ### Grand Finals
+
+  | Group | Sunday, 2018-10-28 |  |  |  |
```

Comment:
Use this formatting instead.

Sunday, 28 October 2018:

| Group | Player 1 |  |  | Player 2 | Match link (if there are any, don't include if it doesn't have all the records) |
| :-: | --: | :-: | :-: | :-- | :-- |